Blazor/wwwroot/cdts-blazor.js

```diff
@@ -0,0 +1,234 @@
+﻿//Get and set the culture name in the local storage
+window.blazorCulture = {
+    get: () => window.localStorage['BlazorCulture'],
+    set: (value) => window.localStorage['BlazorCulture'] = value
+};
+
+var cdtsBlazor = {
+    exitScriptObj: null,
+    globalSerializedFooter: null,
+    globalSerializedTop: null,
+    globalSerializedPreFooter: null,
+    globalSerializedSectionMenu: null,
+    currentPage: null,
+    cdnEnv: null, 
+    defaultCSSHref: "https://www.canada.ca/etc/designs/canada/cdts/gcweb/v5_0_0/cdts/cdts-styles.css",
+
+    //Functions
+    setRefTop: function setRefTop(serializedSetupBase, exitSecureSiteObj, isApp) {
+        this.exitScriptObj = exitSecureSiteObj;
+        wet.localConfig = { cdnEnv: this.cdnEnv, base: { ...serializedSetupBase, isApplication: isApp, sriEnabled: false, cdtsSetupExcludeCSS: true } }; //eslint-disable-line
+        wet.utilities.applyRefTop(() => {
+            wet.utilities.applyRefFooter(() => { 
+                wet.utilities.onRefFooterCompleted();
+            });
+        });
+    },
+
+    setTop: function setTop(serializedTop, isApp) {
+        if (this.globalSerializedTop != serializedTop) {
+            const parsedTop = JSON.parse(serializedTop);
+            const tmpElem = document.createElement('template'); //top's content must be directly in <body>, so use "template" element as temporary container
+            //(can't use outerHTML on an orphan element)
+            tmpElem.insertAdjacentHTML('afterbegin', isApp ? wet.builder.appTop(parsedTop) : wet.builder.top(parsedTop)); //eslint-disable-line
+            //inject a "marker/tag" class
+            for (let e of tmpElem.children) e.classList.add('cdtsblazor-top-tag'); //using `children` and not `childNodes`, we only want elements
+
+            //---[ Remove any elements from previous runs
+            document.body.querySelectorAll('.cdtsblazor-top-tag').forEach((e) => e.remove());
+
+            //---[ Install right after body
+            for (let i = tmpElem.children.length - 1; i >= 0; i--) {
+                const e = tmpElem.children[i];
+                document.body.insertAdjacentElement('afterbegin', e);
+            }
+            
+            if (this.exitScriptObj.exitScript && (this.exitScriptObj.displayModal || this.exitScriptObj.exitURL != "" || this.exitScriptObj.exitURL != null)) this.resetExitScript(this.exitScriptObj);
```

Comment:
(Applies to all such checks) As mention for the possibility of null, theses ifs should start with a `this.exitScriptObj != null &&` condition.

(Side note, since this same if is in a lot of places, it _could_ go be in its down `resetExitScriptForSection` function)

Reply:
done